dotcom-rendering/.storybook/main.ts

```diff
@@ -107,11 +114,12 @@ const webpackConfig = (config: Configuration) => {
 
 	// modify storybook's file-loader rule to avoid conflicts with our svg
 	// https://stackoverflow.com/questions/54292667/react-storybook-svg-failed-to-execute-createelement-on-document
```

Comment:
Is this still relevant?

Reply:
the link to SO? I guess not! I'll remove it